src/components/DisplayProductData.jsx

```diff
@@ -1,20 +1,10 @@
 import React, { Component } from "react";
-import { getData } from "../modules/productData";
 
 class DisplayProductData extends Component {
   state = {
     productData: [],
   };
 
-  componentDidMount() {
-    this.getProductData();
-  }
-
-  async getProductData() {
-    let result = await getData();
-    this.setState({ productData: result });
```

Comment:
How are you updating the date of `productData`when you just moved these functions out of here?